pkg/fakerp/admin_handlers.go

```diff
@@ -60,10 +77,14 @@ func (s *Server) handleListBackups(w http.ResponseWriter, req *http.Request) {
 // handleRestore handles admin requests to restore an etcd cluster from a backup
 func (s *Server) handleRestore(w http.ResponseWriter, req *http.Request) {
 	cs := req.Context().Value(contextKeyContainerService).(*api.OpenShiftManagedCluster)
+	config := req.Context().Value(contextKeyConfig).(*client.Config)
```

Comment:
if ManagementResourceGroup was a pure function of cs.Location and not an additional field in the config, you wouldn't need to do all these gymnastics

Reply:
we use conf for TTL and management resource group name.
This just unifies config placement for deployer function.